plenum/test/txn_author_agreement/test_taa_acceptanse_mechanisms.py

```diff
@@ -0,0 +1,60 @@
+import json
+import pytest
+
+from plenum.common.request import Request
+from plenum.common.constants import TXN_AUTHOR_AGREEMENT_AML, \
+    AML_VERSION, AML, AML_CONTEXT, CURRENT_PROTOCOL_VERSION
+from plenum.common.exceptions import RequestNackedException, RequestRejectedException, InvalidClientRequest
+from plenum.common.util import randomString
+from plenum.test.helper import sdk_get_and_check_replies, sdk_sign_and_submit_req_obj
+
+
+@pytest.fixture(scope="function")
+def taa_aml_request(sdk_wallet_trustee):
+    return Request(identifier=sdk_wallet_trustee[1],
+                   reqId=5,
+                   protocolVersion=CURRENT_PROTOCOL_VERSION,
+                   operation={'type': TXN_AUTHOR_AGREEMENT_AML,
+                              AML_VERSION: randomString(),
+                              AML: json.dumps({'Nice way': 'very good way to accept agreement'}),
+                              AML_CONTEXT: randomString()})
```

Comment:
You can actually use an SDK function here, for example please look at `tests/txn_author_agreement/helpers.py` in latest master

Reply:
Not yet, unfortunatelly, we are waiting for some changes from SDK side